file_organizer_v2/docs/CLI_REFERENCE.md

```
@@ -0,0 +1,246 @@
+# CLI Reference
+
+## Global Options
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)